a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -74,14 +74,20 @@ def _set(self, set_method, c, context, listener):
             except Exception as ex:
                 LOG.exception(ex)
 
-            if bindings and len(bindings) > 0:
+            unbound = any([not bool(x.status) for x in bindings])
+
+            if bindings and len(bindings) > 0 and unbound:
```

Comment:
`if bindings` is too late to do any good, it was already iterated in `unbound = ...`